Add SIMD Support for Java 21

rockfordroeNG opened the pull request:
See: https://github.com/pufferfish-gg/Pufferfish/issues/81#issuecomment-1873195837

rockfordroeNG committed:
Update 0001-Pufferfish-API-Changes.patch

Add support for SIMD for Java 21

rockfordroeNG committed:
Fix minor typo

rockfordroeNG committed:
Update 0001-Pufferfish-Server-Changes.patch

Add support for Java 21 based on this suggestion: https://github.com/pufferfish-gg/Pufferfish/issues/81#issuecomment-1873195837

granny reviewed (changes requested):
Move to the respective `###-Fix-pufferfish-issues.patch` in the `api/` and `server/` patch directories.

Please also PR this to Pufferfish. If Kevin doesn't merge your PR, doesn't update Pufferfish with this change, or doesn't comment on your PR after some weeks pass, then I'll merge this PR.

rockfordroeNG committed:
Move SIMD changes to respective pufferfish fixes

As requested on https://github.com/PurpurMC/Purpur/pull/1499#pullrequestreview-1918950746

rockfordroeNG commented:
Requested changes have been made. A pr to pufferfish has just been filed.

rockfordroeNG committed:
Merge branch 'PurpurMC:ver/1.20.4' into patch-1

rockfordroeNG committed:
Merge branch 'PurpurMC:ver/1.20.4' into patch-1

rockfordroeNG committed:
Merge branch 'PurpurMC:ver/1.20.4' into patch-1

SrBedrock commented:
> Please also PR this to Pufferfish. If Kevin doesn't merge your PR, doesn't update Pufferfish with this change, or doesn't comment on your PR after some weeks pass, then I'll merge this PR.

@granny has enough time passed?

granny committed:
Merge branch 'ver/1.20.4' into patch-1
